src/reference/crash-reporting.md

```diff
@@ -25,19 +25,29 @@ over time. A crash report may contain the following information:
 Google handles all data reported by this tool in accordance with the 
 [Google Privacy Policy][].
 
+Additionally, you may review the recently reported data in the 
+`.dart-tool/dart-flutter-telemetry.log` file under the home directory
+`~/` (macOS/Linux) or the `<username>\AppData` (Windows) directory. 
```

Comment:
**question:** When does this directory get created? I checked my macOS and Windows systems and neither had the file you mention, nor the directory at that location. `.dart-tool` is in each Flutter app directory, but not at the top level directory.

Reply:
This directory along with the files should be getting created in through the latest stable release for Flutter and Dart from i/o. Specifically, we have a new analytics package, `package:unified_analytics` that is responsible for creating these new files.

Once you run anything with the latest flutter cli (ie. `flutter`) or open up VS Code into a dart project (need to have the dart extensions installed), you should see that directory getting created